docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,3 +1,37 @@
-= metallb: A Commodore component to manage metallb
+= MetalLB: A Commodore component to manage MetalLB
 
-{doctitle} is a Commodore component for Managing metallb.
+{doctitle} is a Commodore component for Managing MetalLB.
+The component is based on the https://hub.kubeapps.com/charts/bitnami/metallb[Helm Chart] of Bitnami.
+The Bitnami Helm Chart is https://github.com/metallb/metallb/issues/653[suggested] to become a template for the official Helm Chart of the project.
+
+[CAUTION]
+====
+Currently there is no support of taking over an existing installation executed by the manifests or Kustomize.
+====
+
+
+See the xref:references/parameters.adoc[parameters] reference for further details.
+
+
+== Layer2 Mode (default)
+
+For a minimal configuration providing a VIP is required.
+
+```
+parameters:
+  metallb:
+    addresses:
+      - 10.128.1.200/32
+```
+
+As memberlist key must be generated and stored inside the secret key management.
+
+A pregenerated key ensures, manifests applied multiple times, the secret key does not change unexpectedly lead to failovers, because the speakers can not communicate anymore during a rollover using different secret keys.
```

Comment:
```suggestion
A pregenerated key ensures that the manifests can be applied multiple times without the secret key changing.
Otherwise, elevated fail-over rates may occur during manifest application, as the speakers can not communicate during the rollout, due to differing keys.
```